docs/reference/npm-scripts-and-nx.md

```diff
@@ -44,14 +44,33 @@ For example, if someone were to execute `nx run fxa-auth-server:start`, which st
 These dependencies also work across packages. So if fxa-shared has changed, and we execute `nx run fxa-auth-server:restart`, a `prebuild` and `build` will happen on `fxa-shared`, prior to auth server being restarted!
 
 ## Creating a library in NX
-The following command can be used to create a Nx library in the monorepo: `nx g @nx/node:library <name of library> --directory=<path/to> --buildable` 
-  - e.g. `nx g @nx/node:library capability --directory=libs/payments --buildable` will create a Nx library with the directory `libs/payments/capability`
+
+The following command can be used to create a Nx library in the monorepo: `nx g @nx/node:library <name of library> --directory=<path/to> --buildable`
+
+- e.g. `nx g @nx/node:library capability --directory=libs/payments --buildable` will create a Nx library with the directory `libs/payments/capability`
 
 You can use the `--dry-run` flag to see what will be generated.
 
 After generating the Nx library, you will also need to make the following changes in the library (see other libraries in the monorepo for examples):
-- Rename the job in `project.json` from `test` to `test-unit`
+
+- Rename the job in `project.json` from `test` to `test-unit`, and update the command to run unit tests in the README as well
 - Add `rootDir` in `targets` -> `build` -> `options` of the `project.json`
```

Comment:
Can we include the value of `"rootDir"` here? It is `"."`.

It'd be nice to mention this is stemming from our mixed use of nx integrated libraries and legacy packages in our monorepo and path resolution for nx path aliases: https://mozilla-hub.atlassian.net/browse/FXA-8121. Namely, if a library is used by a package, and this isn't specified, the build command will fail.